metriq_gym/job_manager.py

```diff
@@ -17,6 +16,26 @@ class MetriqGymJob:
     device_name: str
     dispatch_time: datetime
 
+    def to_table_row(self) -> list[str]:
+        return [
+            self.id,
+            self.provider_name,
+            self.device_name,
+            self.job_type,
+            self.dispatch_time.isoformat(),
+        ]
+
+    def serialize(self) -> str:
+        return json.dumps(asdict(self), sort_keys=True, default=str)
+
+    @staticmethod
+    def deserialize(data: str) -> "MetriqGymJob":
```

Comment:
As of Python 3.12+ you can do `MetriqGymJob` over `"MetriqGymJob"`.

Reply:
my IDE doesn't like that though